frontend/src/app/components/containers/complaints/details/complaint-details-create.tsx

```diff
@@ -1096,7 +1097,7 @@ export const CreateComplaint: FC = () => {
                 className="comp-details-label-input-pair"
                 id="reported-pair-id"
               >
-                <label>Reported By</label>
+                <label>Organization reporting the complaint</label>
```

Comment:
See Sonar Cloud issue.   This label should have an htmlFor on it.   

I'm not sure why the incident Date/Time field wasn't also flagged... it probably should as well.